### PR TITLE
8.0 apis tools rename

### PR DESCRIPTION
**Pull request by christinaausley**
## What is the purpose of the change

APIs/Clients -> APIs/Tools rename for 8.0 docs. @pepopowitz Should I also make this change in the 3.8.0 version of Optimize, or would you recommend handling Optimize adjustments in a separate PR?

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by akeller — COMMENTED** [review]
[Super similar to the comment I left on 8.1] 

Building this locally looks and works like I expect, however I want to mention the following:

* The top nav still says "APIs & Clients."
  * I assume that's intended in this PR.
* The sidebar and overview page title still say APIs & Clients
  * Feels like this should be changed in this PR(?)
* The URL is http://localhost:3001/docs/8.0/apis-tools/working-with-apis-tools/, as intended

I clicked around in Optimize docs and didn't see anything unexpected.

File: static/.htaccess [hunk 1]
```diff
@@ -10,7 +10,7 @@ Options -Indexes -MultiViews
 
 # 8.0 apis/clients to apis/tools change
 RewriteRule ^docs/8.0/apis-clients(.*)$ /docs/8.0/apis-tools$1 [R=301,L]
-RewriteRule ^optimize/3.8.0/apis-clients/(.*)$ /optimize/3.8.0/apis-tools$1 [R=301,L]
+RewriteRule ^optimize/3.8.0/apis-clients(.*)$ /optimize/3.8.0/apis-tools$1 [R=301,L]
```
Comment (pepopowitz) [inline static/.htaccess:13]: @christinaausley the "match" on the left side of the rule included a `/` after apis-clients; but the target did not include the `/`. So a URL like `/optimize/3.8.0/apis-clients/optimize-api` was being redirected to `/optimize/3.8.0/apis-toolsoptimize-api/`, causing the broken links.

**Review by akeller — DISMISSED** [review]
:shipit:

**Comment by akeller** [comment]
@christinaausley should I take another look at this and leave a review or do you have more changes you are working on?

**Pull request merged**